### DISCUSSION TITLE
Translation of LaserGrbl to PT-BR

### DISCUSSION THREAD

**Discussion by Tribo-2dot0** [arkypita/LaserGRBL#2121] [General]
I would like to contribute with the PT-BR translation of the software, because some parts already translated contain some errors of agreement, terms not well translated and, now in the latest versions, many things without translation... but I don't know how to access and change the files of the texts.
Would it be possible?

**Comment by ristraus** [comment]
Eu já estou fazendo a tradução. Eu revisei, mas alguns arquivos não foram carregados na última versão.

**Reply by Tribo-2dot0** [reply]
ok... se quiser ajuda é só dizer.

**Reply by ristraus** [reply]
Se quiser revisar a tradução, por favor entre em contato comigo. Pode ser mensagem no Facebook ou no meu bio do instagram @ristraus

**Reply by Tribo-2dot0** [reply]
não sei muito bem achar pessoas no FB... 😁
seria você Richard Strauss, de Salto?

**Comment by ristraus** [comment]
Sim sou eu!

Também estou no instagram como ristraus.

Att

Richard

Em 6 de set. de 2023, à(s) 12:56, Tribo-2dot0 ***@***.***> escreveu:

﻿

não sei muito bem achar pessoas no FB... 😁
seria você Richard Strauss, de Salto?
